celery/bin/celery.py

```diff
@@ -107,9 +109,9 @@ def celery(ctx, app, broker, result_backend, loader, config, workdir,
     ctx.obj = CLIContext(app=app, no_color=no_color, workdir=workdir, quiet=quiet)
 
     # User options
-    worker.params.extend(ctx.obj.app.user_options.get('worker', []))
-    beat.params.extend(ctx.obj.app.user_options.get('beat', []))
-    events.params.extend(ctx.obj.app.user_options.get('events', []))
+    # worker.params.extend(ctx.obj.app.user_options.get('worker', []))
```

Comment:
We need this to support preload options.

Reply:
a compensation for that is done here:
https://github.com/celery/celery/pull/6322/files#diff-a6e64fe5a7c24a0e587e8c92e83b87b5R321

(of course, it is only a draft as it needs to be applied for beat and events as well. i wanted to get an approval for this idea before pursuing it)

there is a downside to this approach, but i'm not sure how else can to get the use options only after the patching